linux/library/flutter_embedder.cc

```diff
@@ -0,0 +1,199 @@
+// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+#include <chrono>
+#include <iostream>
+
+#include <GL/gl.h>
+
+#include <assert.h>
+#include <embedder.h>
```

Comment:
Should this be "embedder.h" rather than <embedder.h> since this isn't a system library?

Also, it should be in a separate block at the end per https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes

Reply:
Fixed!  Also got rid of a few unused include declarations.